subprojects/packagefiles/imguizmo/meson.build

```diff
@@ -0,0 +1,13 @@
+project('ImGuizmo', 'cpp', version: '1.83')
+
+sources = files(
+        'GraphEditor.cpp',
+        'ImCurveEdit.cpp',
+        'ImGuizmo.cpp',
+        'ImSequencer.cpp')
+
+imgui = dependency('imgui')
+
+imguizmo_inc = include_directories('.')
+imguizmo_lib = library('ImGuizmo', sources, include_directories: imguizmo_inc, dependencies: imgui)
```

Comment:
Library names are typically all lowercase unless upstream explicitly chooses to use something else (in this case they don't even build a library file FWICT).